doc/reference/reference_lua/other.rst

```diff
@@ -140,59 +140,101 @@ and variables which are outside all modules.
+
+        tarantool init.lua
+
+    ... the ``/home/testuser/myapp`` path is added to search paths for modules automatically.
+    This means that to load the ``foo`` and ``modules.bar`` modules in ``init.lua``, you only need to add the corresponding ``require`` directives:
+
+    .. code-block:: lua
+
+        -- init.lua --
+        require('foo')
+        require('modules.bar')
+
+    To load modules placed in a non-default directory, use :ref:`setsearchroot() <other-package_setsearchroot>`.
+
 
 .. _other-package_cpath:
 
 .. data:: package.cpath
 
-    This is a string that Tarantool uses to search for C modules,
-    especially important for ``require()``.
-    See :ref:`Modules, rocks and applications <app_server-modules>`.
+    Get a path used by Tarantool to search for C :ref:`modules <app_server-modules>`.
+    For example, these directories are used to find modules loaded using ``require()``.
 
 .. _other-package_loaded:
 
 .. data:: package.loaded
 
-    This is a string that shows what Lua or C modules Tarantool
-    has loaded, so that their functions and members are available.
+    Show Lua or C modules loaded by Tarantool, so that their functions and members are available.
     Initially it has all the pre-loaded modules, which don't need
     ``require()``.
 
 .. _other-package_setsearchroot:
 
 .. function:: package.setsearchroot([search-root])
 
-    Set the search root. The search root is the root directory from
-    which dependencies are loaded.
+    Set the search root, which defines the path to the root directory from which dependencies are loaded.
+    By default, the search root is the current directory.
+
+    :param string search-root: the path to the search root
 
-    :param string search-root: the path. Default = current directory.
+    The ``search-root`` argument value should be a relative or absolute path.
+    If ``search-root`` is a relative path, it is expanded to an absolute path.
 
-    The search-root string must contain a relative or absolute path.
-    If it is a relative path, then it will be expanded to an
-    absolute path.
-    If search-root is omitted, or is box.NULL, then the search root
-    is reset to the current directory, which is found with debug.sourcedir().
+    If ``search-root`` is omitted or set to ``box.NULL``, the search root is reset to the current directory.
 
-    Example:
+    .. NOTE::
+
+        The current directory is obtained using :ref:`debug.sourcedir() <debug-sourcedir>`.
+
+    **Example**
+
+    Suppose external modules are stored outside the application directory, for example:
+
+    .. code-block:: none
+
+        /home/testuser/
+        ├── myapp
+        │   └── init.lua
+        └── .rocks/share/tarantool/
+            └── foo.lua
+
+    In this case, you can specify the ``/home/testuser`` path as the search root for modules in the following way:
+
+    .. code-block:: lua
+
+        -- init.lua --
+        package.setsearchroot('/home/testuser')
```

Comment:
`.rocks` directory is searched recursively upward. This example should work even without the `setsearchroot()` call.